test/unit/logic/community-places.spec.ts

```diff
@@ -38,6 +38,16 @@ describe('Community Places Component', () => {
     mockUserAddress = '0x1234567890123456789012345678901234567890'
     mockCommunityRoles = createMockCommunityRolesComponent({})
     mockPlacesApi = createPlacesApiAdapterMockComponent({}) as jest.Mocked<IPlacesApiComponent>
+    
+    // Re-setup mockLogs after creating mocks
+    mockLogs.getLogger.mockReturnValueOnce({
+      log: jest.fn(),
+      debug: jest.fn(),
+      error: jest.fn(),
+      info: jest.fn(),
+      warn: jest.fn()
+    })
```

Comment:
Would you mind exposing a `createMockLogs` function in the `mocks/component/logs.ts` file if we need to redefine the mock functions?

Reply:
Sure thing! I'll refactor 👼